src/popup/components/TransactionTagList.vue

```diff
@@ -50,11 +50,18 @@ export default defineComponent({
     dense: Boolean,
   },
   setup(props, { root }) {
+    const getTxType = computed(() => root.$store.getters.getTxType);
+    const tx = (
+      getTxType.value(props.tx) === TX_TYPE_MDW.GAMetaTx
+      || getTxType.value(props.tx) === TX_TYPE_MDW.PayingForTx
+    )
+      ? props.tx.tx?.tx as any : props.tx as any;
```

Comment:
The same code was used in [TransactionOverview.vue](https://github.com/aeternity/superhero-wallet/pull/1822/files#diff-7555767fde85bbc840175b6d9e1eaa7e9459580fe4cc884750809f322c9986e2)
So please consider the comment I've made there. And also if the same code is there maybe having an utility function would be better?